src/include/miopen/solver.hpp

```diff
@@ -45,6 +45,14 @@
 
 namespace miopen {
 
+namespace debug {
+
+/// If set to true, then always enable ConvDirectNaive* solver, regardless of environment value
+/// Currently used during driver using naive kernel as gpu reference.
+extern bool AlwaysEnableConvDirectNaive;
```

Comment:
[Recommendation] Put comment about MT environments.

Reply:
comments updated about the environment value name.

Reply:
The commend should say that the use of this debug variable is multi-threaded (MT) contexts may produce unspecified results (btw MIOpenDriver does not use it in MT contexts, so we are safe).